app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
     @author = User.find(params[:user_id])
-    @post = @author.Post
+    @posts = @author.posts
```

Comment:
- [ ] Like the previous reviewer suggested, there is a need to reduce N+1 queries problem by including the comments when you fetch the posts. Take this as a pointer 👉 
```
@posts = @author.posts.includes(:comments)
```

Reply:
@codepantha thanks